src/platform/plugins/shared/console/server/lib/spec_definitions/json/generated/indices.get_alias.json

```diff
@@ -29,7 +29,7 @@
       "{index}/_alias/{name}",
       "{index}/_alias"
     ],
-    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-aliases.html",
+    "documentation": "",
```

Comment:
We should double check that this change is correct

Reply:
Waiting on docs team to get back to me about this change before approving

Reply:
They have created a small PR to fix this issue https://github.com/elastic/elasticsearch-specification/pull/3511/. I'll go ahead and manually update the missing docs for `indices.get_alias` and approve the PR